<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreaksDialog</class>
 <widget class="QDialog" name="BreaksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Remove System Breaks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="intervalButton">
         <property name="accessibleName">
          <string>Break systems every X measures</string>
         </property>
         <property name="text">
          <string>Break systems every</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intervalBox">
         <property name="accessibleName">
          <string>Number of measures</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>measures</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="lockButton">
       <property name="accessibleName">
        <string>Add system breaks at end of each system</string>
       </property>
       <property name="text">
        <string>Add system break at end of each system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="removeButton">
       <property name="accessibleName">
        <string>Remove current system breaks</string>
       </property>
       <property name="text">
        <string>Remove current system breaks</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>intervalButton</tabstop>
  <tabstop>intervalBox</tabstop>
  <tabstop>lockButton</tabstop>
  <tabstop>removeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BreaksDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BreaksDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>754</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
